SECURITY.md

```diff
@@ -0,0 +1,34 @@
+# Security
+
+As part of our [Coordinated Vulnerability Disclosure
+Policy](https://tendermint.com/security), we operate a bug bounty.
+See the policy for more details on submissions and rewards.
+
+The following is a list of examples of the kinds of bugs we're most interested in for
+the Cosmos SDK. See [here](https://github.com/tendermint/tendermint/blob/master/SECURITY.md) for vulnerabilities we are interested in for Tendermint, and lower-level libraries, e.g. IAVL.
+
+## Specification
+- Conceptual flaws
+- Ambiguities, inconsistencies, or incorrect statements
+- Mis-match between specification and implementation of any component
```

Comment:
so... yes, but we're not ready to open up for this yet, they are plenty of currently known inconsistencies.

Reply:
Good point! I'll just remove that section for now, and we can include that again once we get closer to our final feature freeze.